src/wrapper.jl

```diff
@@ -665,10 +665,6 @@ function LGBM_BoosterPredictForMat(
     alloc_len = LGBM_BoosterCalcNumPredict(bst, nrow, predict_type, start_iteration, num_iteration)
     out_result = Array{Cdouble}(undef, alloc_len)
 
-    parameter = ""  # full prediction, no early stopping
```

Comment:
Given the previous comment this is not needed. `num_threads` is always specified as a param (either with a default or customised) and now the additionally there is an option for prediction early stopping.

Reply:
Given the  second comment regarding number of threads, I've refactored this function and removed the `num_threads` as the number of threads will be informed through the `parameter` parameter.